src/models/Expense.model.js

```diff
@@ -1,9 +1,41 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.FLOAT,
```

Comment:
Consider using `DataTypes.DECIMAL` instead of `DataTypes.FLOAT` for the `amount` field to avoid precision issues with floating-point numbers, especially if dealing with currency.